lib/dynflow/action.rb

```diff
@@ -89,6 +89,13 @@ def to_s_humanized
       end
     end
 
+    DelayedEvent = Algebrick.type do
+      fields! execution_plan_id: String,
+              step_id:           Integer,
+              event:             Object,
+              time:              type { variants Numeric, NilClass }
```

Comment:
:warning: On the action class I've removed option to pass `Time` only a number, do you think it's ok?